apidump/summary.go

```diff
@@ -265,11 +265,17 @@ func (s *Summary) printHostHighlights(top *client_telemetry.PacketCountSummary)
 // any packets, capturing packets but failing to parse them, etc.
 func (s *Summary) PrintWarnings() {
 	// Report on recoverable error counts during trace
-	if pcap.CountNilAssemblerContext > 0 || pcap.CountNilAssemblerContextAfterParse > 0 || pcap.CountBadAssemblerContextType > 0 {
-		printer.Stderr.Infof("Detected packet assembly context problems during capture: %v empty, %v bad type, %v empty after parse. ",
+	if pcap.CountNilAssemblerContext > 0 ||
+		pcap.CountNilAssemblerContextAfterParse > 0 ||
+		pcap.CountBadAssemblerContextType > 0 ||
+		pcap.CountZeroValuePacketTimestamp > 0 ||
+		pcap.CountLastPacketBeforeFirstPacket > 0 {
+		printer.Stderr.Infof("Detected packet assembly context problems during capture: %v empty, %v bad type, %v empty after parse, %v zero timestamp, %v last before first timestamp. ",
```

Comment:
This is good but we should figure out some more K8s-friendly way of showing our counters, for example including them in telemetry or dumping them out periodically to the log.

Reply:
I'll do this in a follow up PR, I have to first update the schema in the shared libs repo before I can push these values through the telemetry worker.